src/reducers/removeIdenticalDataFields.js

```diff
@@ -34,40 +34,15 @@ function isRelevantField8(field) {
   return field.subfields.some(sf => getSubfield8Value(sf) !== undefined);
 }
 
-function isUnlinkedDataField(field) {
-  return field.tag !== '880' && field.subfields && !isRelevantField6(field) && !isRelevantField8(field);
-}
-
-
-function recordGetAllSubfield8Indexes(record) {
-  /* eslint-disable */
-  let subfield8Values = [];
-  record.fields.forEach(field => {
-    if (!field.subfields) {
-      return;
-    }
-    field.subfields.forEach(sf => {
-      const index = getSubfield8Index(sf);
-      if (index > 0 && !subfield8Values.includes(index)) {
-        //nvdebug(`Add subfield \$8 ${index} to seen values list`, debugDev);
-        subfield8Values.push(index);
-      }
-    });
-  });
-
-  return subfield8Values;
-  /* eslint-enable */
+function getSubfield8Value(subfield) {
```

Comment:
just noting whether the alternative of having a filter function for subfield8 (or better yet, a generalized filter that accepts requested subfield as parameter) would be sufficient alternative so that .map could be utilized for gathering the value?

Reply:
Also technically the isValidSubfield8 can be used as filtering function when iterating over subfields.